app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -651,7 +651,9 @@ form#update_profile_form {
   .submit_block { margin-bottom: 20px; }
 }
 
-select#user_language, select#user_color_theme, #user_auto_follow_back_aspect_id, #aspect_ids_ {
+select#user_language,
```

Comment:
- Avoid using id selectors
- Avoid qualifying id selectors with an element.
- Selector `user_language` should be written in lowercase with hyphens
- Selector `user_color_theme` should be written in lowercase with hyphens
- Selector `user_auto_follow_back_aspect_id` should be written in lowercase with hyphens